scapy/contrib/tzsp.uts

```diff
@@ -0,0 +1,503 @@
+from enum import test
+% TZSP test campaign
+
+#
+# execute test:
+# > test/run_tests -P "load_contrib('tzsp')" -t scapy/contrib/tzsp.uts
+#
+
++ syntax check
+= check if the import works
+
+from scapy.contrib.tzsp import *
```

Comment:
This is not needed as we run the tests with the appropriate option to load the contrib module.

Reply:
but this gives a clean indication if something is wrong with the imported file - but I will remove it for the sake of consistency.